src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4367,6 +4423,25 @@ public override bool PreVisit(CallNode node)
                         startArg++;
                     }
 
+                    if (overloadsWithTypeArgs.Any() && node.Args.Count > 1)
+                    {
+                        var nodeInp = node.Args.Children[0];
```

Comment:
Suggestion: `nodeInput` - no need to abbreviate here.

Reply:
runs in redeclaration issue issue as nodeInput is declared later in this method. thats why even the if block above has nodeInp